Resources/docs/adapters/awss3.md

```diff
@@ -13,7 +44,33 @@ Adapter for Amazon S3 SDK v2.
    This directory will be created in the root of the bucket and all files will be read and written there.
    * `acl` Default ACL to apply to the objects
 
-## Defining services
+Also note that when you create a bucket it [is located in a specific region](http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html). 
```

Comment:
Can you remove this line and the one below please (they're already [here](https://github.com/KnpLabs/KnpGaufretteBundle/pull/193/files#diff-bd830abb396dedb7f711473d5371dd96R33)). After that it'll be OK to merge ;)

Reply:
done in a separate commit, are you fine with it?
I could squash my commits if you want

Reply:
Yes it would be nice if you could squash them :)

Reply:
done :)